samples/nearby-chat/gradle.properties

```diff
@@ -0,0 +1,21 @@
+# Project-wide Gradle settings.
```

Comment:
Was there a particular need for a specific gradle.properties file for this library? The root file has everything we need and it's in one place to control the properties for all the components.

Reply:
No. It was copied and pasted. I have removed it without ill effect.